dev/scripts/src/alluxio.org/build/cmd/build.go

```diff
@@ -158,8 +184,11 @@ func TarballF(args []string) error {
 	if err := os.RemoveAll(tarballPath); err != nil {
 		return stacktrace.Propagate(err, "error deleting path %v", tarballPath)
 	}
+	if err := os.MkdirAll(filepath.Dir(tarballPath), os.ModePerm); err != nil {
```

Comment:
these change are to accommodate custom tarball output locations? originally i only planned to allow the name to be configured, not the directory (thus hardcoding to repoRoot). double checking that this is the case